lib/iuliia/schema.rb

```diff
@@ -28,6 +27,8 @@ def schemas
 
       def load_schema(name)
         JSON.parse(File.read("lib/schemas/#{name}.json"), object_class: OpenStruct, symbolize_names: true)
+      rescue Errno::ENOENT => e
+        puts "Failed to load schema. #{e.message}"
```

Comment:
Honestly, I think it will be better to throw `Exception`, maybe even custom class exception. Thing is, wrong schema name is quite critical event. No schema - no work, full stop.

But original exception is not fully understandable for a random gem user. So we need to give him clear message: check your schema name, you did it wrong.

Reply:
now user get 'Specified schema does not exist' message and the gem raises the exception:

```
irb(main):001:0> Iuliia::Schema['gost_779a']
Traceback (most recent call last):
        5: from bin/console:15:in `<main>'
        4: from (irb):1
        3: from /home/theendcomplete/Documents/projects/OS/iuliia-rb/lib/iuliia/schema.rb:10:in `[]'
        2: from /home/theendcomplete/Documents/projects/OS/iuliia-rb/lib/iuliia/schema.rb:25:in `block in schemas'
        1: from /home/theendcomplete/Documents/projects/OS/iuliia-rb/lib/iuliia/schema.rb:30:in `load_schema'
Iuliia::Exceptions::NonExistentSchemaException (Specified schema does not exist)
irb(main):002:0> 
```